data/mapcss-mapping.csv

```diff
@@ -255,7 +255,7 @@ waterway|ditch|tunnel;[waterway=ditch][tunnel?];;name;int_name;254;
 deprecated|deprecated;255;x
 piste:type|downhill|intermediate;[piste:type=downhill][piste:difficulty=intermediate];;name;int_name;256;
 railway|subway_entrance;257;
-deprecated|deprecated;258;x
+amenity|charging_station|bicycle;[amenity=charging_station][bicycle?];;name;int_name;258;
```

Comment:
1. maybe `small_electric_vehicle=yes` should be considered same as `bicycle=yes`?
2. maybe make priority of `amenity-charging_station-motorcar` a little higher for charging stations with `[bicycle=yes],[motorcar=yes]`?

Reply:
1. why?

Reply:
I assume its the same charger for e-bikes, electric scooters, skate boards, etc.. Probably its just a power outlet?